react/ToastContext.js

```diff
@@ -0,0 +1,3 @@
+import { ToastContext } from './components/ToastProvider/index'
+
+export default ToastContext
```

Comment:
Just check if this breaks while linking the app

Reply:
Done :grin: